README.md

```diff
@@ -166,38 +156,40 @@ The Hazelcast configuration to use DNS Lookup looks as follows.
 
 ```xml
 <hazelcast>
-  <properties>
-    <property name="hazelcast.discovery.enabled">true</property>
-  </properties>
-
   <network>
     <join>
-      <!-- deactivate normal discovery -->
       <multicast enabled="false"/>
-      <tcp-ip enabled="false" />
-
-      <!-- activate the Kubernetes plugin -->
-      <discovery-strategies>
-        <discovery-strategy enabled="true"
-            class="com.hazelcast.kubernetes.HazelcastKubernetesDiscoveryStrategy">
-
-          <properties>
-            <property name="service-dns">MY-SERVICE-DNS-NAME</property>
-            <property name="service-dns-timeout">10</property>
-          </properties>
-        </discovery-strategy>
-      </discovery-strategies>
+      <kubernetes enabled="true">
+        <service-dns>MY-SERVICE-DNS-NAME</service-dns>
+      </kubernetes>
     </join>
   </network>
 </hazelcast>
 ```
 
+```java
+config.getNetworkConfig().getJoin().getMulticastConfig().setEnabled(false);
+config.getNetworkConfig().getJoin().getKubernetesConfig().setEnabled(true)
+      .setProperty("service-dns", "MY-SERVICE-DNS-NAME");
+```
+
 There are 2 properties to configure the plugin:
  * `service-dns` (required): service DNS, usually in the form of `SERVICE-NAME.NAMESPACE.svc.cluster.local`
  * `service-dns-time` (optional): custom time for how long the DNS Lookup is checked
 
 **Note**: In this README, only XML configurations are presented, however you can achieve exactly the same effect using Java-based configurations.
 
+## Scaling Hazelcast cluster in Kubernetes
+
+Hazelcast cluster is easily scalable within Kubernetes. You can use the standard `kubectl scale` command to change the cluster size.
+
+Note however that, by default, Hazelcast does not shutdown gracefully. It means that if you suddenly scale down by more than your `backup-count` property (1 by default), you may lose the cluster data. To prevent that from happening, set the following properties:
+- `terminationGracePeriodSeconds`:  in your StatefulSet (or Deployment) configuration; the value should be high enough to cover the data migration process
+- `-Dhazelcast.shutdownhook.policy=GRACEFUL`: in the JVM parameters
+- `-Dhazelcast.graceful.shutdown.max.wait`: in the JVM parameters; the value should be high enough to cover the data migration process
+
+The graceful shutdown configuration is already included in Hazelcast Helm Charts.
+
```

Comment:
What do you think about promoting helm chart here because those settings done by default?

Reply:
Ok, added a link.